docs/tutorial.rst

```diff
@@ -1645,10 +1646,11 @@ keywords such as ``{GREEN}`` or ``{BOLD_BLUE}``.  Colors have the form shown bel
   can use!
 
 You can make use of additional variables beyond these by adding them to the
-``PROMPT_FIELDS`` environment variable.  The values in this dictionary
-should be strings (which will be inserted into the prompt verbatim), or
-functions of no arguments (which will be called each time the prompt is
-generated, and the results of those calls will be inserted into the prompt).
+``PROMPT_FIELDS`` environment variable. This is also how you set the ``time_format``
+variable. The values in this dictionary should be strings (which will be inserted
```

Comment:
seems like this is over emphasizing on time_format field. I think this part of change is not needed.

Reply:
Removed